_getting-started/index.md

```diff
@@ -4,21 +4,32 @@ title: Getting Started
 partof: getting-started
 languages: [ja]
 includeTOC: true
+scala3: true
 
-redirect_from: "/getting-started.html"
+redirect_from:
+  - "/getting-started.html"
+  - "/scala3/getting-started.html"
```

Comment:
The old URL, https://docs.scala-lang.org/scala3/getting-started.html will still work (it will redirect to /getting-started).